### ISSUE TITLE
Mekanism: Version 9

### ISSUE THREAD

**Issue by R1ZL4** [F32Organization/Faithful32-1.7.10#545]
Version 9 has been released with its own new textures, none of the F32 textures seem to work.

**Comment by xMrVizzy** [comment]
Okay! I'll try)

**Comment by mcdodge34** [comment]
I can confirm, and it seems to cause problems with some machines like the Electrolytic Separator, its now transparent kind of very annoying.

**Comment by Xireka** [comment]
Delete the mekanism folder from the texture pack folder, fixes the issue, even if it's kind of a temporary fix. :)

**Comment by mcdodge34** [comment]
Hum, I tried to fix the issue by finding the machine textures in the folder and deleting only them, I don't know what to remove to fix it totally, I was sure I had them all but I still have a messed up Electrolytic Separator, I wanna keep the other stuff like ores, dusts, ingots, blocks, armors and tools, those work well still, its just the machines that are messed up badly.

**Comment by StefanTT** [comment]
I will have a look at the textures

**Comment by StefanTT** [comment]
@R1ZL4 please have a look if your problem is fixed now

**Comment by mcdodge34** [comment]
Will take a look at them tomorrow late at night.

**Comment by mcdodge34** [comment]
Seems the Electrolytic Separator textures are fixed, but the ones for the electric pump aren't yet.

**Comment by StefanTT** [comment]
You are right, the electric pump texture is still there. I will remove it tonight (in 12 hours).

**Comment by mcdodge34** [comment]
Thanks, doing awesome work, will report if I find anything else, I'm playing with this mod for the 2nd time only, the new textures that aidan did for the machines are awesome I know they're supposed to blend into the Unity Texture Pack, but I don't really like the pack, everyone has different taste I suppose.

**Comment by StefanTT** [comment]
@mcdodge34 the issue should be fixed by now, can you please verify it?

**Comment by mcdodge34** [comment]
Will gladly take a look today, I'll report back for sure.

**Comment by StefanTT** [comment]
The pack from f32.me is regularily updated. I think it's daily, but I do not know the schedule.

**Comment by mcdodge34** [comment]
I can't really use symbolic links cause I have a lot of personalized textures in there that I took from several different packs, can't distribute it this way but we can use the textures we want for personal use, hope I'm right with this. But all my gamesaves for all my games are using symbolic links to sync to my dropbox account, so I never loose anything.

**Comment by StefanTT** [comment]
You can have another directory with only the personalized textures and add both as resource packs in minecraft.

**Comment by mcdodge34** [comment]
That would be tricky to setup for me, I also removed a bunch of the textures that the modpack I use don't need, I find I gain some FPS and a little bit on the loading time, I take the full pack each time and remove all the textures of the mods that aren't in the pack I play with.

FYI, Electric Pump textures seems to be OK according to NEI crafting grid. No I got to sleep, work at 9am and its already 3am here.
